tests/ClientTest.php

```diff
@@ -173,4 +174,55 @@ public function testSelectDatabasePassesOptions()
         $this->assertInstanceOf(WriteConcern::class, $debug['writeConcern']);
         $this->assertSame(WriteConcern::MAJORITY, $debug['writeConcern']->getW());
     }
+
+    public function testCreateClientEncryption()
+    {
+        $client = new Client(static::getUri());
+
+        $options = [
+            'keyVaultNamespace' => 'default.keys',
+            'kmsProviders' => ['aws' => (object) ['accessKeyId' => 'abc', 'secretAccessKey' => 'def']],
```

Comment:
Same question as https://github.com/mongodb/mongo-php-library/pull/708#discussion_r369545255. Is the `(object)` cast needed?